src/components/JoinGroupForm/JoinGroupForm.tsx

```diff
@@ -127,13 +180,17 @@ export default function JoinGroupForm() {
         <>
           <div className={styles['class-info-container']}>
             <h2 className={styles['class-info-title']}>Tu grupo:</h2>
-            <h3 className={styles['class-name']}>{classInfo.class_name}</h3>
+            <h3 className={styles['class-name']}>{classInfo.subject_name}</h3>
             <span className={styles['class-label']}>Profesor/a</span>
-            <p className={styles['class-info']}>{classInfo.teacher}</p>
+            <p
+              className={styles['class-info']}
+            >{`${classInfo['teacher.first_name']} ${classInfo['teacher.last_name']}`}</p>
             <span className={styles['class-label']}>Días </span>
-            <p className={styles['class-info']}>{classInfo.days}</p>
+            <p className={styles['class-info']}>{classInfo.days.toString()}</p>
             <span className={styles['class-label']}>Horario</span>
-            <p className={styles['class-info']}>{classInfo.time}</p>
+            <p
```

Comment:
As I watched your demo, I was a bit confused with the time. I even thought there was a bug.
I think it was mainly because I saw it in a format hr:min:sec, and the start and end times were too close.
Maybe we could give it a different format? Seconds are not important to the user under this context.
Also, I think maybe a bit more of css could improve the user experience

Reply:
The time format comes from the backend. What I can do is to trim the string to include only up to minutes